### DISCUSSION TITLE
Add link in README to talk in Japanese?

### DISCUSSION THREAD

**Discussion by joelnitta** [ropensci/targets#835] [Ideas]
Recently [I gave a talk on targets](https://togotv.dbcls.jp/20220331.html) for the [Bio"Pack"athon](https://sites.google.com/view/biopackathon) meetup group in Japanese.

@wlandau [I have a patch with an addition to the README](https://github.com/joelnitta/targets/tree/patch-1#%E6%97%A5%E6%9C%AC%E8%AA%9E) if you think this would be a useful addition. Please let me know.

**Comment by wlandau** [comment]
Yes please, I would really appreciate a PR! Thank you for spreading the word about `targets`.

**Reply by joelnitta** [reply]
Great! Thank you for the fantastic package :)

[PR](https://github.com/ropensci/targets/pull/842) submitted (sorry for the delay!)
